sklearn/linear_model/tests/test_ransac.py

```diff
@@ -341,6 +344,9 @@ def test_ransac_min_n_samples():
     with pytest.raises(ValueError):
         ransac_estimator7.fit(X, y)
 
+    with pytest.raises(ValueError):
```

Comment:
While the other test are not doing it, it would be best to add the (partial) message that we expect to raise

```suggestion
    err_msg = "A value for min_samples must be provided"
    with pytest.raises(ValueError, match=err_msg):
```